src/components/global/logo/logo.hbs

```diff
@@ -1,5 +1,5 @@
 <a class="logo" href="/">
-	<img class="logo-img" src="/img/logo/adg.svg" alt="Logo">
+  <svg class="logo-img" viewBox="0 0 33 44" role="img" aria-labelledby="title"><title>ADG Logo</title><path fill="currentColor" fill-rule="evenodd" d="M11.87,16.07H6L4.84,20.16h-4L7.27.56H10.7L17,20.16H13ZM6.92,12.85H11L9,5.64ZM18.7,20.16V.56h5.07a9.07,9.07,0,0,1,3.53.67A7.71,7.71,0,0,1,30,3.09a8.4,8.4,0,0,1,1.73,2.84,10.42,10.42,0,0,1,.61,3.64v1.61a10.32,10.32,0,0,1-.61,3.63A8.62,8.62,0,0,1,30,17.65a7.77,7.77,0,0,1-2.65,1.85,8.69,8.69,0,0,1-3.42.66ZM22.5,3.63V17.12H24a4.31,4.31,0,0,0,1.9-.41,3.93,3.93,0,0,0,1.43-1.17,5.36,5.36,0,0,0,.89-1.86,9.1,9.1,0,0,0,.31-2.5V9.54a9,9,0,0,0-.3-2.4,5.34,5.34,0,0,0-.9-1.87,4.23,4.23,0,0,0-1.48-1.21,4.57,4.57,0,0,0-2-.43ZM15.71,41.29a5.35,5.35,0,0,1-1,.91,7.53,7.53,0,0,1-1.5.82,10.05,10.05,0,0,1-1.94.6A11.33,11.33,0,0,1,9,43.85a7.48,7.48,0,0,1-3-.59,6.39,6.39,0,0,1-2.3-1.71,7.82,7.82,0,0,1-1.48-2.74,12.26,12.26,0,0,1-.51-3.68V32.47a11.58,11.58,0,0,1,.54-3.68A8.23,8.23,0,0,1,3.7,26,6.51,6.51,0,0,1,6,24.31a6.75,6.75,0,0,1,2.82-.59,8.92,8.92,0,0,1,2.88.42,5.41,5.41,0,0,1,3.41,3.12,8.53,8.53,0,0,1,.62,2.54H12a4,4,0,0,0-.91-2.27,3,3,0,0,0-2.23-.76,2.62,2.62,0,0,0-1.34.37,3.22,3.22,0,0,0-1.08,1.09A5.79,5.79,0,0,0,5.72,30a9.89,9.89,0,0,0-.27,2.44v2.69a8.19,8.19,0,0,0,.92,4.21,3.05,3.05,0,0,0,2.76,1.47,5,5,0,0,0,.87-.07,4.8,4.8,0,0,0,.81-.19,4.67,4.67,0,0,0,.67-.27,1.78,1.78,0,0,0,.43-.33l0-3.55H8.68V33.57h7ZM24.36,35l-5.27-1.29L20,31,25,33.19l-.48-5.8h2.94L27,33.29l5-2.2.89,2.82L27.5,35.13l3.6,4.35-2.39,1.73-2.84-5L23,41.07l-2.41-1.66Z"/></svg>
```

Comment:
I will check whether we can include this during the build so we don't have to duplicate it. But let's merge it anyway.